src/serverdlg.cpp

```diff
@@ -72,17 +72,17 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
                                          "registration failed, please choose another server list." ) );
 
     // custom directory server address
-    QString strCentrServAddr = "<b>" + tr ( "Custom Directory Server Address" ) + ":</b> " +
-                               tr ( "The custom directory server address is the IP address or URL of the directory "
-                                    "server at which the server list of the connection dialog is managed." );
+    QString strCustomDirectoryAddress = "<b>" + tr ( "Custom Directory Server Address" ) + ":</b> " +
+                                        tr ( "The custom directory server address is the IP address or URL of the directory "
+                                             "server at which the server list of the connection dialog is managed." );
 
-    lblCentralServerAddress->setWhatsThis ( strCentrServAddr );
-    edtCentralServerAddress->setWhatsThis ( strCentrServAddr );
-    edtCentralServerAddress->setAccessibleName ( tr ( "Directory server address line edit" ) );
+    lblDirectoryAddress->setWhatsThis ( strCustomDirectoryAddress );
+    edtDirectoryAddress->setWhatsThis ( strCustomDirectoryAddress );
+    edtDirectoryAddress->setAccessibleName ( tr ( "Directory server address line edit" ) );
 
-    cbxCentServAddrType->setWhatsThis ( "<b>" + tr ( "Server List Selection" ) + ":</b> " +
-                                        tr ( "Selects the server list (i.e. directory server address) in which your server will be added." ) );
-    cbxCentServAddrType->setAccessibleName ( tr ( "Server list selection combo box" ) );
+    cbxDirectoryType->setWhatsThis ( "<b>" + tr ( "Server List Selection" ) + ":</b> " +
```

Comment:
Suggest "Directory Server Selection"?

Reply:
I left a lot of this Directory Type stuff until later:
https://github.com/pljones/jamulus/compare/patch/ocd-reorder-of-serverdlg-bits...pljones:feature/server-list-persistence-ui?expand=1#diff-8e8a36ea447bef8b1c8e5cfe4f3d19662a8ee937333f46fede7f9f6e64b6f575R55-R72

Reply:
OK, cool. So would it help if I go ahead and approve this PR as it is now? Although I'm not sure who gets to do the second approval at the moment, for the C++ stuff.

Reply:
> src/connectdlg.cpp:48 - change "Use the List dropdown" to "Use the Directory dropdown".
> src/connectdlg.cpp:74-75 - reword to use "directory" terminology.

Those two need catch, though.

Reply:
OK, as they're in the translations and I wanted this to avoid any UI text, I've created a separate branch, dependent upon this one:
https://github.com/pljones/jamulus/commit/f7a8b181d958b003df2efc9be6b1fc2f944538e9

Reply:
(That commit id is now stale)